### PR TITLE
Add Watermelon Moscow Mule

### PR DESCRIPTION
**Pull request by Sina-Sche**
Fix small bugs in my json file, hopefully all is going to work now 👍

### CODE REVIEW

File: src/recipes/watermelon-moscow-mule.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            "ingredient": "Vodka"
+        },
+        {
+            "quantity": "6",
+            "measure": "oz",
+            "ingredient": "Ginger Beer"
+        },
+        {"quantity": "",
+        "measure": "",
+        "ingredient": "Mint leaves"}
```
Comment (alfg) [inline src/recipes/watermelon-moscow-mule.json:23]:
```suggestion
        "ingredient": "Mint leaves"},
```

File: src/recipes/watermelon-moscow-mule.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        
+        {"quantity": "",
+        "measure": "",
+        "ingredient": "Ice cubes"}
```
Comment (alfg) [inline src/recipes/watermelon-moscow-mule.json:27]:
```suggestion
        "ingredient": "Ice cubes"},
```

File: src/recipes/watermelon-moscow-mule.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        {"quantity": "",
+            "measure": "",
+            "ingredient": "Slice of lime"}
+    ],
+    "directions": [
+        "Slice watermelon, blend and strain",
+        "Add Watermelon, Vodka and ice into shaker and shake for 30 seconds",
+        "Pour over ice in mule mug and top it off with ginger beer. Add mint leaves and a slice of lime and enjoy!"
+    ],
+    "image": "<watermelon-moscow-mule.jpg>",
```
Comment (alfg) [inline src/recipes/watermelon-moscow-mule.json:37]:
```suggestion
    "image": "watermelon-moscow-mule.jpg",
```

File: src/recipes/watermelon-moscow-mule.json [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    "keywords": [
+        "classic",
+        "alcoholic",
+        "refreshing",
+        "ginger beer",
+        "vodka",
```
Comment (alfg) [inline src/recipes/watermelon-moscow-mule.json:43]:
```suggestion
        "vodka"
```

**Review by alfg — CHANGES_REQUESTED** [review]
Added a few suggested fixes. Thanks.

**Comment by Sina-Sche** [comment]
> Added a few suggested fixes. Thanks.

Thanks a lot!!!

**Comment by alfg** [comment]
Looks good. Thanks for fixing!

**Pull request merged**